draft-ietf-rtcweb-jsep.xml

```diff
@@ -1513,7 +1515,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               new ICE credentials, for the purpose of forcing an ICE
               restart and kicking off a new gathering phase, in which
               the new servers will be used. If the ICE candidate pool
-              has a nonzero size, any existing candidates will be
+              has a nonzero size, and a non-provisional local description
+              has not yet been applied, any existing candidates will be
```

Comment:
So you'd prefer that leftover candidates can still be used? Cullen and my reasoning was: "if you can't resize the pool mid-session, it makes sense to go a step further and say the pool isn't used at all mid-session".

Also, without the ability to lower the pool size, this would mean that if you pooled more candidates than you'll ever need, there's *no* way to discard them.